client/me/billing-history/style.scss

```diff
@@ -65,6 +65,18 @@
 	&:last-child {
 		border-bottom: none;
 	}
+
+	&.is-placeholder {
+		height: auto;
+
+		td {
```

Comment:
Not blocking, but CSS Specificity is getting a touch high here. (0 id,2 class,1 element). The current class names here won't allow us to do it, but in the future we could pull this out into so that specificity remains at (0,2,0) or (0,1,0)

For example something similar to:

(0,2,0)
```
.billing-history__data.is-placeholder {
    padding: 24px;
}
```

(0,1,0)
```
.billing-history__placeholder-data {
    padding: 24px;
}
```